app/javascript/packages/step-indicator/step-indicator-element.ts

```diff
@@ -0,0 +1,80 @@
+const SMALL_VIEWPORT_MEDIA_QUERY = '(max-width: 639px)';
```

Comment:
Unfortunately, git/GitHub didn't pick up the diff as being a lift-and-shift. This file is pretty much identical to the previous `index.ts`, with the exception (addition) of these lines:

https://github.com/18F/identity-idp/blob/8c01ee1bca7893badbc6f728381a4793c6546cb4/app/javascript/packages/step-indicator/step-indicator-element.ts#L70-L78